cmd/release-controller/http.go

```diff
@@ -52,6 +52,7 @@ const htmlPageEnd = `
 
 const releasePageHtml = `
 <h1>Release Status</h1>
+<div class="alert alert-warning">This site is part of OpenShift's continuous delivery pipeline. Neither the builds linked here nor the upgrade paths tested here are officially supported. Please visit the Red Hat Customer Portal for the latest supported product details.</div>
```

Comment:
```html
...Please visit <a href="https://access.redhat.com/downloads/content/290/">the Red Hat Customer Portal</a> for...
```

for these hunks?

Reply:
I can't verify the link's content, because I dont have an active subscription, but I'll take your word for it! ;)

![RC5](https://user-images.githubusercontent.com/43015045/95263070-3efebe00-07fb-11eb-9ce4-0056ea9222fe.png)

Reply:
Yeah that's a decent URL, thanks.

Reply:
Looks like we dropped the link based on "[folks will just know this][1]", although personally I have to hunt around a bit to find that link and would have preferred something clickable.

[1]: https://github.com/openshift/release-controller/pull/218#discussion_r502409777